lib/pagseguro/subscription/subscription_search.rb

```diff
@@ -0,0 +1,135 @@
+module PagSeguro
+  class SubscriptionSearch
+    # Set the report options.
+    #
+    # # +per_page+: the page size.
+    # # +starts_at+: the report's starting date. Can't be older than 6 months.
+    # # +ends_at+: the report's ending date. Can't be greater than 30 days from the starting date.
+    #
+    attr_reader :options
+
+    # Set the errors from the report request.
+    attr_reader :errors
+
+    # Return the current page.
+    attr_reader :page
+
+    def initialize(path, options)
+      @path = path
+      @page = options.delete(:page) || 0
```

Comment:
Ok, if it is this project's default (and not from Pagseguro itself) let's write it down to change :metal: